.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
The VSCode extension that integrates with prettier requires there to be a config file in order to pull prettier out of the repo (instead of looking for a globally installed version), but we don't actually want to apply any configuration options, so we can just leave it empty.